exercises/paasio/description.md

```diff
@@ -9,4 +9,4 @@ statistics. The wrapper must report:
 - The total number of bytes read/written.
 - The total number of read/write operations.
 
-[PaaS]: http://en.wikipedia.org/wiki/Platform_as_a_service
+[paas]: http://en.wikipedia.org/wiki/Platform_as_a_service
```

Comment:
potentially breaks a link

Reply:
It does. Same issue:

```markdown
[PaaS][]
```

should have been

```markdown
[PaaS][paas]
```